file/options.go

```diff
@@ -11,10 +11,12 @@ type Options struct {
 	// Path is the path to the folder / directory to begin traversing.
 	Path string `arg:"" optional:"" name:"path" help:"The path to the folder to begin traversing" env:"GUM_FILE_PATH"`
 	// Cursor is the character to display in front of the current selected items.
-	Cursor    string `short:"c" help:"The cursor character" default:">" env:"GUM_FILE_CURSOR"`
-	All       bool   `short:"a" help:"Show hidden and 'dot' files" default:"false" env:"GUM_FILE_ALL"`
-	File      bool   `help:"Allow files selection" default:"true" env:"GUM_FILE_FILE"`
-	Directory bool   `help:"Allow directories selection" default:"false" env:"GUM_FILE_DIRECTORY"`
+	Cursor      string `short:"c" help:"The cursor character" default:">" env:"GUM_FILE_CURSOR"`
+	All         bool   `short:"a" help:"Show hidden and 'dot' files" default:"false" env:"GUM_FILE_ALL"`
+	Permissions bool   `short:"p" help:"Show file permissions" default:"false" env:"GUM_FILE_PERMISSION"`
+	Size		bool   `short:"s" help:"Show file size" default:"false" env:"GUM_FILE_SIZE"`
```

Comment:
I think we should default to true, what do you think? Then we can add the `negatable:""` option to allow for people to do `--no-permissions` and `--no-size`.

Reply:
Yes indeed, that way default would remain as is.